src/instrumenter/cha.ts

```diff
@@ -179,16 +176,16 @@ export function getCHA(srcs: SourceUnit[]): CHA<ContractDefinition> {
     }
 
     const res = { parents, children, roots, leaves };
-    assert(checkCHA(res), `Built CHA ${res} is malformed`);
+
+    assert(checkCHA(res), "Built CHA {0} is malformed", ppCHA(res));
```

Comment:
`ppCHA` is probably fairly expensive. Maybe move this inside an `if (!checkCHA(res)) { assert(false, "...", ppCHA(res)); }?

Reply:
It would be easier to `throw` than `assert()` in such case. Will fix.

Reply:
Addressed in https://github.com/ConsenSys/scribble/pull/94/commits/d4ed33eadbc26a96e9b7719cff33f967783af572